lib/galaxy/tools/filters/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Deprecated module for toolbox filters.
 
 Filters placed in this module will still work - but filters should be
-moved to lib/galaxy/tools/toolbox/filters.
```

Comment:
Should we have a copy of this file also in `lib/galaxy/tools/toolbox/filters` ?

Reply:
I was going to wait for someone to complain, but perhaps we should.